aiven-client.spec

```diff
@@ -1,13 +1,24 @@
-Name:           aiven-client
+%global project aiven-client
+
+Name:           %{project}
```

Comment:
note, probably this python package should end up being called python3-aiven-client to be fully fedora python guidelines compatible